File: String/reverse_string/c++/reverse.cpp
#include <iostream>
#include <bits/stdc++.h>
using namespace std;

string reversed(string);

int main(int argc, char** argv){
	cout<<reversed(argv[1])<<endl;
	return 0;
}

string reversed(string s){
	reverse(s.begin(),s.end());
	return s;
}

File: String/reverse_string/javascript/reverse.js
function reversed(s){
	return [...s].reverse().join("")
	
	//alternate ways for string to array 
	//1. s.split('')
	//2. Array.from(s)
	//3. Object([],s)
	//4. [...s] this is so far the best one
	
}
console.log(reversed(process.argv[2]));
